integration/v3_grpc_test.go

```diff
@@ -69,47 +69,93 @@ func (c *clusterV3) RandClient() *clientv3.Client {
 
 // TestV3PutOverwrite puts a key with the v3 api to a random cluster member,
 // overwrites it, then checks that the change was applied.
-func TestV3PutOverwrite(t *testing.T) {
+func TestV3PutOverwriteGRPC(t *testing.T) {
 	defer testutil.AfterTest(t)
+	testV3PutOverwrite(t, false)
+}
+
+func TestV3PutOverwriteClient(t *testing.T) {
```

Comment:
Ok will make the test name consistent and split tests into a separate file. Thanks